system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -470,6 +479,34 @@ pub type AssetFeeAsExistentialDepositMultiplierFeeCharger = AssetFeeAsExistentia
 	TrustBackedAssetsInstance,
 >;
 
+match_types! {
+	pub type SystemParachains: impl Contains<MultiLocation> = {
+		MultiLocation {
+			parents: 1,
+			interior: X1(Parachain(
+				system_parachain::ASSET_HUB_ID |
+				system_parachain::ENCOINTER_ID |
+				system_parachain::BRIDGE_HUB_ID
+			)),
+		}
+	};
+}
+
```

Comment:
We should use the common `ConcreteAssetFromSystem` implementation that's in https://github.com/polkadot-fellows/runtimes/pull/62 so that we don't need to update this across multiple runtimes every time there's a new system para.

Reply:
Sorry, didn't get it - we need `Contains<MultiLocation>` here and `ConcreteAssetFromSystem` is `ContainsPair<MultiAsset, MultiLocation>`. IIUC we could have something similar to `RelayOrOtherSystemParachains` that won't have this generic `SystemParachainMatcher` argument and instead will match on `ParaId::from(*id).is_system()` instead, right? But it's not yet in sdk (and not in `1.3`) - I could add a separate crate, shared by all SP runtimes, here and add this impl there or I could open PR in a sdk - then we could reuse this implementation once it is published

Reply:
Opened https://github.com/paritytech/polkadot-sdk/pull/2422, so I'd wait for sdk release first to use it here, meanwhile leaving it as is. As I said, I could add some temporary crate, shared by all SP here, but then we would need to remove it on next release, which is imo not worth it. But feel free to insist - could do it here too

Reply:
No all good, we can update with the next SDK release.